src/index.js

```diff
@@ -0,0 +1,37 @@
+import './style.css';
+import Task from './modules/task.js';
+
+const taskinput = document.querySelector('.add_todo_input');
+const listContainer = document.querySelector('.todolist');
+const todoaddbutton = document.querySelector('.todo_add_icon');
+const refreshIcon = document.querySelector('.refresh_div');
+const tasklist = [];
+function generateTodoIndex() {
+  let id = 1;
+  if (tasklist.length > 0) {
+    id = tasklist[tasklist.length - 1].index + 1;
+  }
+  return id;
```

Comment:
- [ ] Great job Implementing the add book functionality 👏🏻 However this is not required in this milestone. See 👇🏻 

![18](https://user-images.githubusercontent.com/79058364/192905098-727196e9-c709-4633-b77e-6500a456f7eb.png)

- [ ] According to the project requiement: 
1. You need to create an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\]. 

   ```javascript
   const tasks = [
     {
       description: 'An example ...',
       completed: false,
       index: 10,
     },
          .......
   ];
   ```
2. Create a function that iterates over the tasks array and populates the HTML list item element for each task.

   ```javascript   
   const populateTasks = (task, parentElement) => {
     const { description, completed, index: id } = task;
     const innerHTML = `
            <p>...</p>
     const newTodoListElement = document.createElement('li');
     Object.assign(newTodoListElement, { innerHTML, id });
     parentElement.appendChild ......
   };
   ```
-  Note you don't have to implement this function exactly like this, this is only "one" example of how it can be implemented.

3. On page load, make sure the task array is rendered dynamically to the DOM. (Note)The tasks should appear in order of the `index` values for each task. this means, the first book should have an index of 1 and not zero